tutorials/4 Plugin Developer/Create new QRenderer.ipynb

```diff
@@ -252,22 +278,21 @@
     "    return RouteMeander(design, component_name, myoptions)\n",
     "\n",
     "asym = 90\n",
-    "cpw1 = connect('cpw1', 'Q1', 'd', 'Q2', 'c', '6.0 mm', f'+{asym}um', fillet='25um')\n",
-    "cpw2 = connect('cpw2', 'Q3', 'c', 'Q2', 'a', '6.1 mm', f'-{asym}um', flip=True, fillet='100um')\n",
-    "cpw3 = connect('cpw3', 'Q3', 'a', 'Q4', 'b', '6.0 mm', f'+{asym}um', fillet='75um')\n",
-    "cpw4 = connect('cpw4', 'Q1', 'b', 'Q4', 'd', '6.1 mm', f'-{asym}um', flip=True)\n",
+    "cpw1 = connect('cpw1', 'Q1', 'd', 'Q2', 'c', '5.7 mm', f'+{asym}um', fillet='25um')\n",
+    "cpw2 = connect('cpw2', 'Q3', 'c', 'Q2', 'a', '5.6 mm', f'-{asym}um', flip=True, fillet='100um')\n",
+    "cpw3 = connect('cpw3', 'Q3', 'a', 'Q4', 'b', '5.5 mm', f'+{asym}um', fillet='75um')\n",
+    "cpw4 = connect('cpw4', 'Q1', 'b', 'Q4', 'd', '5.8 mm', f'-{asym}um', flip=True)\n",
     "\n",
     "gui.rebuild()\n",
     "gui.autoscale()"
    ]
   },
   {
-   "cell_type": "code",
-   "execution_count": null,
+   "cell_type": "markdown",
    "metadata": {},
-   "outputs": [],
    "source": [
-    "Headings.h1('Exporting a skeleton file.')"
+    "### Export text to file using your custom QSkeletonRenderer\n",
+    "The QSkeletonRenderer class contains several sample methods. Let's use one intended to print out the content of the QGeometry table to a text file (Remember: QGeometry contains the list of the raw layout shapes that compose the design, which we have created in the previous cell)."
```

Comment:
The QGeometry tables are not printed to file.

Suggest:
Let's use one intended to print out the name of the QGeometry tables to a text file.  The entire QDesign uses three tables.

Reply:
Strange that you appear to have reviewed the older version of the notebook according to github it is the last one. very weird. Anyways, my latest version has the phrasing similar to yous.